k9mail/build.gradle

```diff
@@ -17,6 +17,27 @@ dependencies {
     compile 'com.android.support:support-v13:21.0.2'
     compile 'net.sourceforge.htmlcleaner:htmlcleaner:2.2'
     compile 'de.cketti.library.changelog:ckchangelog:1.2.1'
+
+    androidTestCompile ('com.jakewharton.espresso:espresso:1.1-r3' ) {
```

Comment:
Agreed. I wasn't strict enough in limiting contributions to the other repository to PGP-related changes.
I'm leaning towards abandoning the [k9mail_pgp_mime](https://github.com/k9mail/k9mail_pgp_mime) repository and continuing the PGP work in a branch of the main repository. The idea of a separate repository was to make it easier for new contributers  to join. While there have been contributions from some new people I don't feel like this experiment can be declared a success.